oshi-core/src/main/java/oshi/hardware/platform/unix/freebsd/FreeBsdDisks.java

```diff
@@ -66,6 +66,8 @@ public static boolean updateDiskStats(HWDiskStore diskStore) {
             // In KB
             diskStore.setReadBytes((long) (ParseUtil.parseDoubleOrDefault(split[3], 0d) * 1024));
             diskStore.setWriteBytes((long) (ParseUtil.parseDoubleOrDefault(split[4], 0d) * 1024));
+            // # transactions
+            diskStore.setCurrentQueueLength((long) (ParseUtil.parseDoubleOrDefault(split[5], 0d)));
```

Comment:
This all looks good... I'll merge in the morning. One last fix if you get to it before I do, please use `parseLongOrDefault()` here and in line 112.  I confirmed from the source code this will be a uint64 value.

Reply:
done